docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<blockquote>Matrix profile indicates slided wrong place as discords.</blockquote><p> </p><p>As I explained in my comment above, I think we should rephrase it. We may say: "The discord discovered by matrix profile is not what are looking for."</p><p> </p><p>I am just trying to avoid using the word "wrong" here. In my head, contrast matrix profile is another tool that can be used to reveal something. Wether that thing is what we are loooking for or not depends on our perspective and our goal. In this case, we will probably see that contrast abstract profile can reveal something that is not captured by matrix profile. But, is that always useful? Well, that depends on the domain. In other words, a user does not know if the discord discovered by matrix profile is useful. The same goes for contrast matrix profile. A user can compute both and then investigate their outcome to see which reveals better information about the data.</p><p> </p><blockquote>ost important thing is Matrix profile also doesn't indicate remarkable distance for the discord.</blockquote><p> </p><p>How can one know if it is a remarkable distance for discord or not? I think this sentence is a little bit biased. This is because we already saw the matrix profile computed for the first half of <code>T</code>and we now feel the maximum distance shown in figure above is not large enough. So, I think we should remove this sentence as we prefer to not be subjective in our analysis.</p><p> </p><p> </p><blockquote>=&gt; Because similar anomalies are nearest neighbor relationship.</blockquote><p> </p><p>Maybe we should say this first:</p><p> </p><p>"Because the targets we are looking for are similar to each other, they can be nearest neighbor of each other and hence their corresponding distance in matrix profile will be small. Therefore, none of them will be detected via matrix profile"</p><p> </p><blockquote>We find those similar anomalies from the dataset with those following specification.</blockquote><p> </p><p>Now, I think this is the place that we should do our best and provide a clear explanation. We can say: "To find the desirable anomalies, we should first understand what properties distinguish them from the other patterns. Note that these two anomalies are similar to each other but they are dissimmilar to other (regular) patterns. This is the main concept behind the contrast matrix profile."</p><p> </p><p> </p><blockquote>This is exactly the property we desire.&nbsp;In other words, we need to prepare two following data.</blockquote><p> </p><p>Then you can create a section here and name it "Contrast matrix profile". Then, we say:</p><p>"To compute the contrast matrix profile, two time series data are needed as follows: "</p><p> </p><p> </p><blockquote>Take a look at the plot below to confirm the condictions</blockquote><p> </p><p>Are we showing any important information here? If not, then we can just remove this line and the code you have provided for plotting the two time series <code>T_p</code> and <code>T_n</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I replaced sententce with your suggested contents.</p>